sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1720,4 +1720,11 @@ object functions {
      UnresolvedFunction(udfName, cols.map(_.expr))
   }
 
+  /**
+   * Parsers the comma separated expression strings into the columns they represent, similar to
+   * DataFrame.selectExpr
+   */
+  def expr(expStrs: String*): Seq[Column] = expStrs.map { expr =>
```

Comment:
The function type was changed as per @rxin comment above to be `String -> Column`. The new variable name is `expr`.